config.js

```diff
@@ -215,6 +218,9 @@ module.exports = {
     DEFAULT_PERSIST_REDIS_SAMPLE_STORE_MILLISECONDS,
   port,
   prioritizeJobsFrom,
+  queueStatsActivityLogsInterval,
+  queueTimeMillis95th: pe.QUEUESTATS_95TH_WARNING_MILLIS ||
+    undefined,
   rateLimit,
```

Comment:
pe.QUEUESTATS_95TH_WARNING_MILLIS resolves to "undefined" if not defined. Not sure what the best practice though.